GettingStarted.ipynb

```diff
@@ -54,12 +62,25 @@
     "import os\n",
     "import pathlib\n",
     "import listing\n",
+    "import multiprocessing as mp\n",
     "\n",
-    "xsdirName = 'xsdir'\n",
-    "datapath = pathlib.Path(os.environ['DATAPATH'], xsdirName)\n",
+    "xsdirName = 'xsdir_mcnp6.3'\n",
     "# If your XSDIR file is not called `xsdir` you will need to change the \n",
-    "# xsdirName variable obove to match the name of your file\n",
-    "listing.generateJSON(datapath, 20)"
+    "# xsdirName variable above to match the name of your file\n",
+    "\n",
+    "datapath = pathlib.Path(os.environ['DATAPATH'], xsdirName)\n",
+    "\n",
+    "if not datapath.is_file():\n",
+    "    print(\"ERROR: xsdir file {}\".format(datapath),\"does not exist\")\n",
+    "else:\n",
+    "    print(\"Reading from xsdir at: {}\".format(datapath))\n",
+    "\n",
+    "# The variable N is how many parallel threads are used\n",
+    "# By default, it uses all but one of the available CPUs\n",
+    "    N = max(1, mp.cpu_count()-1)\n",
```

Comment:
Your indentation here is not consistent; it's hard to tell from this interface. Please fix this and then I think It's ready to go. Although, I am still (informally) testing on my end to make sure everything works properly.

Reply:
Ah, you're right. Then please also indent the comments.

Reply:
Will do.  Do you want the default value of ```xsdirName``` set back to ```xsdir```?

Reply:
If ```xsdirName``` is not set here, the cell crashes with a ```name xsdirName is not defined``` error.

Reply:
For now, just leave it as `xsdir`. I'll make some changes in a future Pull Request.